### ISSUE TITLE
Bold/italics asciidoc symbols within word should not be escaped

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/reverse_adoc#70]
<img width="851" alt="Screen Shot 2020-07-03 at 11 30 00 PM" src="https://user-images.githubusercontent.com/11865/86482732-1d69e280-bd85-11ea-993c-c24da76b9783.png">

From:
```adoc
=== class of activity

class that has only term:[individual activity] as members

[example]
====
'Distilling' is a class of activity that is reference data held in a reference data library. The classification of the individual activity 'Distill batch\_27 on 2006-05-19' as 'distilling' specifies what it is.
====

[example]
====
'Distillation mode D\_3' is a class of activity that is an operating mode. Information can be recorded which defines the class of activity 'Distillation mode D\_3'. The classification of the individual activity 'Distill batch\_27 on 2006-05-19' as a member of 'Distillation mode D\_3' specifies what it is.
====

[example]
====
'Activities within maintenance contract JB\_2006' is a class of activity that is defined by enumerating its members. The classification of the individual activity 'remove debris from site' as a member of 'Activities within maintenance contract JB\_2006' specifies that it shall carried out as part of the contract.
====

[.source]
<<ISO_10303-221_2006,clause=3.1>>
```

In AsciiDoc, only when the bold/italic symbol is at a word boundary it is considered a bold/italic symbol (`this is _bold_`). When it is located within a word, you need to "double it" (`this is not_bold, but this is__very__bold`).

A slash in the second case (e.g. `Distill batch_27 on 2006-05-19`) will actually render as `Distill batch\_27 on 2006-05-19`.

I think this might be an issue of ReverseAdoc. If that's the case, we should move this issue over there.

This applies to the bold (`*`), italics (`_`) symbols.

ping @w00lf

**Comment by ronaldtse** [comment]
This is a challenge in AsciiDoc: https://discuss.asciidoctor.org/Difficulty-in-consistently-escaping-markup-td5318.html

**Comment by w00lf** [comment]
@ronaldtse i have fixed udnerscore(italic) char escape inside word - https://github.com/metanorma/reverse_adoc/pull/71. I can easily add the same code for `*` char but in that case cases with the first bold char will break, eg: `*B*old first char`.

**Comment by opoudjis** [comment]
But `*` is completely analogous to `_` in Asciidoc, so whatever you do to the one should be done to the other. (Same applies for ` in fact.)

So a boldface A*B should be rendered as `*A*B*`: it will be instances of double `**` in a word that will be difficult, and that will force use of a workaround such as a document attribute.

**Comment by ronaldtse** [comment]
@w00lf what @opoudjis means is that the treatment for `_` should also apply for `*`. Let's do the same treatment. Thanks.

**Comment by w00lf** [comment]
> @w00lf what @opoudjis means is that the treatment for `_` should also apply for `*`. Let's do the same treatment. Thanks.

Implemented in https://github.com/metanorma/reverse_adoc/pull/72

**Comment by ronaldtse** [comment]
Released as 0.3.1.
